lib/components/blog.dart

```diff
@@ -330,6 +351,96 @@ class ListItem extends StatelessWidget {
  * navigation links. Navigation links collapse into
  * a hamburger menu on screens smaller than 400px.
  */
+
+List<HeaderItem> headerItems = [
+  HeaderItem(title: 'HOME', onTap: () {}),
+  HeaderItem(title: 'PORTFOLIO', onTap: () {}),
+  HeaderItem(title: 'STYLE', onTap: () {}),
+  HeaderItem(title: 'ABOUT', onTap: () {}),
+  HeaderItem(title: 'CONTACT', onTap: () {}),
+];
+
+Widget headerLogo(BuildContext context) {
```

Comment:
Doubt: 
Why use help methods in this case instead of widgets? 
I think it would be nice to follow best practices, as the [Flutter team recommends](https://www.youtube.com/watch?v=IOyq-eTRhvo&t=1s).